### ISSUE TITLE
The label for Veteran ID needs to be read by AT

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12099]
When viewing a case, the Veteran ID is displayed. Assistive technology only reads the case number. The label Veteran ID as well as the number should be read by assistive technology programs. (New: 08/22/2019)  the labels need to be read with AT

Suggested fix: Add label to the Veteran ID button to indicate clicking on it copies the ID

**Comment by yoomlam** [comment]
Useful ARIA usage: https://developer.paciellogroup.com/blog/2018/09/describing-aria-describedby/

Solution: 
1. adding `aria-label` will cause AT to read the atttribute value.  Using `aria-labelledby="vIdLabel"` will only read "Veteran ID"
2. adding `describedby`will cause AT to read the referenced element's content (after `aria-label` is read).

```
        <li data-css-p21ib3="" id="vIdLabel" >Veteran ID:&nbsp;
        	<span data-tip="true" data-for="tooltip-500000018" currentitem="false">
        		<button type="submit" class="cf-apppeal-id" data-css-1u97fj0="" aria-describedby="tooltip-500000018" aria-label="Veteran ID 500000018">500000018&nbsp;

        			<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21 21">
                        <path d="M13.346 2.578h-2.664v-1.29C10.682.585 10.08 0 9.35 0H6.66c-.728 0-1.33.584-1.33 1.29v1.288H2.663v2.577h10.682V2.578zm-4.02 0H6.66v-1.29h2.665v1.29zm6.685 3.89V3.234a.665.665 0 0 0-.678-.656H14v1.29h.68v2.576H6.66v9.046H1.333V3.867h.68v-1.29H.678a.665.665 0 0 0-.68.657v12.913c0 .365.302.656.68.656h6.006v3.867h9.35l3.996-3.867V6.468h-4.02zm0 12.378v-2.043h2.112l-2.11 2.043zm2.665-3.356H14.68v3.867H7.992v-11.6h10.682v7.733z" fill="#5B616B" fill-rule="evenodd"></path>
                    </svg></button>
            </span>
            <span data-css-1l3lo7h="">
                <div class="__react_component_tooltip place-bottom type-dark " id="tooltip-500000018" data-id="tooltip" style="left: 180px; top: 281px;">Click to copy</div>
            </span></li>
        <li data-css-p21ib3=""><span data-css-dw3c54=""><a>View all cases</a></span></li>
```
